components/server/src/iam/iam-session-app.spec.ts

```diff
@@ -196,7 +196,7 @@ class TestIamSessionApp {
         expect(result.body?.message).to.equal("OIDC client config id is missing");
     }
 
-    @test.only public async testSessionRequest_createUser_removes_admin() {
```

Comment:
@AlexTugarev was this a leftover from developing?

Reply:
nice catch!
thanks!